### PR TITLE
Auto driving and use pose estimator

### PR DESCRIPTION
**Pull request by prensing**
This is the final version we tested in the shed, and works pretty well. We should pull it into main for all future work.

### CODE REVIEW

**Comment by coachFitz0807** [comment]
Looks good.

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 1]
```diff
@@ -180,11 +162,42 @@ public void drive(ChassisSpeeds chassisSpeeds) {
...
+    return m_kinematics;
+  }
+
+  public void stop() {
+    drive(new ChassisSpeeds(0, 0, 0));
+  }
+
   @Override
   public void periodic() {
+    System.out.println("*** DriveTrain PERIODIC ***");
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:189]: This line should be commented out. We don't want to print this every loop.

**Review by coachFitz0807 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java (hunk 1 above)
Comment (prensing) [reply src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java]: I just merged in swerveTraj just to be safe, and that is now gone. At the time, we were not sure it was getting called.

**Comment by coachFitz0807** [comment]
I'm good now.

**Pull request merged**